crates/threshold-bls/src/sig/blind.rs

```diff
@@ -85,113 +62,59 @@ where
 
         // r * H(m)
         // XXX result from zexe API but it shouldn't
-        h.map(msg).unwrap();
+        h.map(msg).expect("could not map to the group");
```

Comment:
Should we change to return a Result?

Reply:
Yeah breaking changes in the APIs which will affect FFI etc. will be done in a separate PR